src/cf-core/src/cf-base.less

```diff
@@ -70,6 +70,11 @@ body {
     line-height: @base-line-height;
 }
 
+strong,
+b {
+    font-weight: 600;
+}
```

Comment:
Could you please move this below the `button, input, select, textarea` rule? I'd like to keep that rule together with the `body` rule because of the unusual nature of having to set `font-family` twice.